package.json

```diff
@@ -46,6 +46,28 @@
     "test": "./node_modules/.bin/karma start ./karma.conf.js",
     "travis": "./node_modules/grunt-cli/bin/grunt ci --verbose --force & npm run test"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
They do in theory, but npm has has had issues before. That said, I think we can wait to see if anything breaks, as it really shouldn't, and we do want patch updates if any arise.